index.html

```diff
@@ -82,194 +115,210 @@ <h2>
     </section>
     <section>
       <h2>
-        Registry table of powerful features
+        Registry table of standardized permissions
       </h2>
       <p>
-        [=Powerful features=] are identified by a unique string token, which is used for
-        integration with the [[[Permissions]]] and [[[Permissions-Policy]]] specifications.
+        For a permission to appear in the table of standardized permissions, and thus be considered
+        a <dfn>standardized permission</dfn>, it needs to meet the following criteria:
+      </p>
+      <ul>
+        <li>Implemented and demonstrably interoperable in at least two browser engines (e.g., has
+        accompanying <a href="https://web-platform-tests.org">Web Platform Tests</a>).
+        </li>
+        <li>Is specified in published as [=technical report=] by a W3C [=Working Group=] with
+        maturity of [=first public working draft=] or above, or is published by the <a href=
+        "https://whatwg.org">WHATWG</a> as a standard.
+        </li>
+      </ul>
+      <p>
+        Each [=permissions=] is identified by a unique <dfn data-dfn-for="permission">string
```

Comment:
```suggestion
        Each [=permission=] is identified by a unique <dfn data-dfn-for="permission">string
```

Reply:
Up above you talk about permissions policies as "token string"s... but here they're "string tokens". Is that difference intentional?

Reply:
Nope, good catch!